README.md

```diff
@@ -68,22 +68,36 @@ An example of a simple randomized cocotb testbench:
 # test_dff.py
 
 import random
+
 import cocotb
 from cocotb.clock import Clock
-from cocotb.triggers import FallingEdge
+from cocotb.triggers import RisingEdge
+from cocotb.types import LogicArray
 
 @cocotb.test()
-async def test_dff_simple(dut):
-    """ Test that d propagates to q """
+async def dff_simple_test(dut):
+    """Test that d propagates to q"""
+
+    # Set initial input value to prevent it from floating
+    dut.d.setimmediatevalue(1)
+    # Assert initial output is unknown
+    assert LogicArray(dut.q.value) == LogicArray("X")
```

Comment:
Could potentially leave this out of the `README.md` to make the example more concise?

Reply:
I think it's good as is.